app/views/manage_outcomes/outcomes/_outcomes.html.erb

```diff
@@ -13,7 +13,13 @@
         <td><%= render "progress_bar",
           amount: outcome.active_assessments_with_results_count,
           total: outcome.active_assessments_count %></td>
-        <td><%= link_to t(".edit_outcome"), edit_manage_outcomes_outcome_path(outcome) %></td>
+        <td>
+          <%= link_to t(".edit_outcome"), edit_manage_outcomes_outcome_path(outcome) %>
+
+          <% if outcome.outcome_assessments.any? %>
+            <%= link_to t(".view_assessments"), manage_assessments_course_assessments_path(@course, outcome_ids: outcome) %>
```

Comment:
@derekprior Using this to filter when coming from a link. I noticed when you filter direcrtly on the page you get `?utf8=✓&outcome_ids=2`. Is the utf param something I should include here as well?